src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -83,16 +120,147 @@ If you are using a device:
   Where the hex string is your device’s (or emulator’s) code. This means adb has found your device (or emulator).
 
 {% endcapture %}
-{% include modules/column-w-toc.liquid
+{% include modules/one-column.liquid
   id: "set-up-your-computer-and-android-emulator-or-device"
-  content: content_with_toc
+  content: content
+%}
+
+<!-- END: Single Column Body Module -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Check for Firefox for Android compatibility
+::: note alert
+This section only applies to Firefox for Android < 68 (Fennec)
+:::
+
+Before running your extension on Firefox for Android, consider using [`web-ext lint`](/documentation/develop/web-ext-command-reference#web-ext-lint). Lint performs a check to determine if any of the permissions, manifest keys, and web extension APIs you’re using are incompatible with Firefox for Android. Lint relies on your extension’s manifest.json file including `strict_min_version`, it then reports on the features that are not supported by the minimum version you have set.
+
+In the lint report:
+
+- incompatible permissions are identified by `PERMISSION_FIREFOX_ANDROID_UNSUPPORTED_BY_MIN_VERSION`,
+- incompatible manifest keys are identified by `KEY_FIREFOX_ANDROID_UNSUPPORTED_BY_MIN_VERSION`, and
+- incompatible APIs are identified by `ANDROID_INCOMPATIBLE_API`
+
+similar to this:
+
+![linter screenshot](/assets/img/documentation/develop/linter_output2.png)
+
+::: note alert
+Lint does not report on APIs that are not implemented by Firefox or Firefox for Android.
+:::
+
+When setting `strict_min_version`, unless you’re targeting a specific version of Firefox, choose the most recent version of Firefox you expect your extension to be compatible with. For example, you can reasonably expect that most installations of Firefox for Android will be the current or previous version. So, if the current version is 66, consider setting 65 is the minimum version:
+
+```json
+"browser_specific_settings": {
+  "gecko": {
+    "strict_min_version": "65.0"
+  }
+}
+```
+
+{% endcapture %}
+{% include modules/one-column.liquid
+  id: "check-for-firefox-for-android-compatibility"
+  content: content
+%}
+
+<!-- END: Single Column Body Module -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Install and run your extension in Firefox for Android
+::: note alert
+This section only applies to Firefox for Android < 68 (Fennec)
+:::
+
+In your extension, ensure that you've included an application ID using the [browser_specific_settings](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) key in the manifest.json:
+
+```json
+"browser_specific_settings": {
+  "gecko": {
+    "id": "borderify@example.com"
+  }
+}
```

Comment:
This (an explicit id in the manifest) would still be necessary while using web-ext on Fenix